src/doc/using_visit/SavingPrinting/Saving_movies.rst

```diff
@@ -190,11 +139,251 @@ click the **Movie progress dialog's** **Cancel** button.
+   Reflected Curve Overlay 2D/3D plot options
+
+The options for the Curve plot portion are the same as used by the **Curve Overlay** template.
+
+The rest of the Save Movie wizard pages for the **Reflected Curve Overlay** template are the same as for all movies described previously.
+
+.. _reflected_curveoverlay_frame:
+
+.. figure:: images/reflected_curveoverlay_frame.png 
+   :scale: 75%
+   
+   Frame from a Reflected Curve Overlay template movie
+
+
+.. _Slice3DTemplate:
+
+Slice 3D Movie Template
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The **Slice 3D** movie template animates a slice plane through all three dimensions of a 3D dataset before fading to black.
+
+.. _slice3d_preview:
+
+.. figure:: images/slice3d_preview.png 
+   :scale: 75%
+
+   Slice 3D template preview.
+
+
+The next page (:numref:`Figure %s<slice3d_options>`) lets you fill in details such as which dataset will be sliced.
+You can also set visualization title and colors.
+
+.. _slice3d_options:
+
+.. figure:: images/slice3d_options.png 
+   :scale: 75%
+
+   Slice 3D parameters.
+
+This movie template ends with a fade to black sequence.
+You can set the duration of this sequence as well as the colors used as shown in :numref:`Figure %s<slice3d_fade_options>`.
+   
+.. _slice3d_fade_options:
+
+.. figure:: images/slice3d_fade_options.png 
+   :scale: 75%
+
+   Slice 3D fade sequence options.
+
+
+Creating new Templates
+""""""""""""""""""""""
+
+.. _savemovie_createtemplate:
+
+.. figure:: images/savemovie_createtemplate.png 
+   :scale: 75%
+   
+   Creating a new template
+
+.. _SidebySideMovie:
+
+Side by Side Movie
+^^^^^^^^^^^^^^^^^^
+To make a side by side movie you can use the **Create template** option.
+Before saving the movie with this template, you must set up the plots that you want to make in multiple vis windows, as shown in :numref:`Figure %s<sidebyside_plots>`.
+
+.. _sidebyside_plots:
+
+.. figure:: images/sidebyside_plots.png 
+   :scale: 75%
+   
+   Setting up plots for Side by Side movie generation.
+
+Since you are creating a new movie template, you first must set up the movie viewports.
+You can create many viewports and the viewports can be adjacent or can overlap (e.g. picture in picture).
+When viewports overlap, you can make the top viewports semi-transparent, make a background color transparent, or even add drop shadows.
+There are some existing layouts if you don't want to get fancy.
+Select the **Side by Side** layout and VisIt will create 2 viewports next to one another.
+
+.. _sidebyside_layout_selection:
+
+.. figure:: images/sidebyside_layout_selection.png 
+   :scale: 75%
+   
+   Selecting Side by Side layout.
+
+.. _sidebyside_viewports:
+
+.. figure:: images/sidebyside_viewports.png 
+   :scale: 75%
+   
+   The Side by Side viewports.
+
+After you create movie viewports, you must map sequences of images to the viewports.
+Sequences are series of images and they can be produced from the plots in your vis windows or transitions such as fades, etc.
```

Comment:
```suggestion
After you create movie viewports, you must map sequences of images to the viewports.
These can be produced from the plots in your vis windows or transitions such as fades, etc.
```

I thought it was confusing to say that sequences of images are series of images.